skeletons/constr_SET_OF_jer.c

```diff
@@ -64,11 +63,6 @@ SET_OF_encode_jer(const asn_TYPE_descriptor_t *td, const void *sptr, int ilevel,
 
     if(!sptr) ASN__ENCODE_FAILED;
 
-    if(xcan) {
-        encs = (jer_tmp_enc_t *)MALLOC(list->count * sizeof(encs[0]));
-        if(!encs) ASN__ENCODE_FAILED;
-        cb = SET_OF_encode_jer_callback;
```

Comment:
This is the only usage of `SET_OF_encode_jer_callback`
Is that something that shouldn't exist, or is it being behind `if(0)` a mistake?

Reply:
It looks like it was a way to change cb for the call to elm->type->op->jer_encoder on line 85, and then on line 108 it's set back to the original, but never used again?